sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
+        if "vti" in str(device) and "ip_vti0" not in str(device):
```

Comment:
agree, using "ip_" could be more flexible and avoid modification in case. Updated. please review. thanks.